features/home/src/main/res/values-ro-rRO/strings.xml

```diff
@@ -3,4 +3,7 @@
     <string name="default_label">Teacher Workout</string>
     <string name="profile">Profile screen</string>
     <string name="home_landing">Home landing screen</string>
+    <string name="themes_title">Categorii de lecții</string>
```

Comment:
Thanks for helping to keep the app bilingual. :+1:

Reply:
Didn't notice the stuff outside my code. Tunnel vision. Haha. I know it was late at night. But what translations would you suggest though?

Reply:
They are good translations. Was just happy you add them.